include/swift/Basic/Dwarf.h

```diff
@@ -21,7 +21,7 @@
 
 namespace swift {
   /// The DWARF version emitted by the Swift compiler.
-  const unsigned DWARFVersion = 3;
+  const unsigned DWARFVersion = 4;
```

Comment:
I see.  Let's see what @adrian-prantl says, possibly this change would need to be scoped to Cygwin.